app/views/shopping_carts/show.html.haml

```diff
@@ -0,0 +1,17 @@
+%h2 Your Shopping Cart
+%table.shopping-cart
+  %tr
+    %th Name
+    %th Price
+  - @cart.shopping_cart_items.each do |cart_item|
+    %tr
```

Comment:
small thing we need to add a css to the th tag and aline left to make it lock better vi kan make a shore for it in PT